### DISCUSSION TITLE
Maximum 15 products per category?

### DISCUSSION THREAD

**Discussion by j-toner** [swellstores/community#471] [Help] [answered]
I can only return 15 products per category using
 ```
await swell.products.list({
					category: category
				});
``` 

I'm still on the community plan is this why?

**Comment by ericingram** [comment, marked as answer]
I think that's just the default. Try adding `limit: 50` (or whatever) to your parameters

**Reply by j-toner** [reply]
Wow it worked! I could have sworn I already tried that...🤦  Thanks for the quick response!
